deployEnvironment.sh

```diff
@@ -0,0 +1,13 @@
+cd src/Docker
+docker-compose down
+git fetch
+git checkout --force $1
+
+cd ..
+chmod +x buildVersionFiles.sh
+./buildVersionFiles.sh
```

Comment:
We'll need to add the `generateTokenKey` bit in here too.

Reply:
This is going to take some time as I need to restructure this script so secrets aren't written into checked in files. I'd like to completely get rid of the `.env.production` file as it's causing a lot of issues, probably the `.env.local` one too.

Reply:
I've now updated this in the configuration for production so it uses an environment variable for the system. This means I don't need to run the script during deployment.